packages/core/src/node/logger-cli-contribution.ts

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (C) 2018 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as yargs from 'yargs';
+import { injectable } from 'inversify';
+import { LogLevel } from '../common/logger';
+import { CliContribution } from './cli';
+import * as fsextra from 'fs-extra';
+import * as nsfw from 'nsfw';
+import { Event, Emitter } from '../common/event';
+import * as path from 'path';
+
+/** Convert string to LogLevel, throw if invalid.  */
+function readLogLevelString(levelStr: string, errMessagePrefix: string): LogLevel {
```

Comment:
How about to make it a method? https://github.com/theia-ide/theia/wiki/Coding-Guidelines#di-function-export

Reply:
Done.